src/ticketmap/index.tsx

```diff
@@ -61,6 +69,28 @@ interface TicketGroupsBySectionByZone {
   }
 }
 
+const createDeepEqualSelector = createSelectorCreator(defaultMemoize, isEqual)
+
+const $missingSectionIds = createDeepEqualSelector(
+  (state: State) => state.ticketGroups,
+  (state: State) => state.sectionZoneMapping,
+  (ticketGroups, sectionZoneMapping) => ticketGroups
+    .map(ticketGroup => ticketGroup.tevo_section_name.toLowerCase())
+    .filter(sectionId => sectionZoneMapping[sectionId] === undefined)
+)
+
+const $availableTicketGroups = createDeepEqualSelector(
+  (state: State) => state.ticketGroups,
+  (state: State) => state.sectionZoneMapping,
+  (ticketGroups, sectionZoneMapping) => ticketGroups
+    .map(ticketGroup => ({
+      section: ticketGroup.tevo_section_name,
+      zone: sectionZoneMapping[ticketGroup.tevo_section_name.toLowerCase()],
+      price: ticketGroup.retail_price
+    } as NormalizedTicketGroup))
```

Comment:
I don't believe you need to cast this object, you could type it by defining the return value for the function
```typescript
.map((ticketGroup: TicketGroup): NormalizedTicketGroup => ({
```

Reply:
Oh, right on!